lib/rubycritic/core/rating.rb

```diff
@@ -22,6 +22,16 @@ def to_h
       @letter
     end
 
+    def color
+      case @letter
+      when "A" then :green
+      when "B" then :green
+      when "C" then :yellow
+      when "D" then :light_red
+      when "F" then :red
+      end
```

Comment:
Let's rather use a constant pointing to a hash like this:

```Ruby
RatingToColor = {
  A: :green,
  # ....
}
```

This will express our intent clearer (after all, we're talking about a constant mapping), will provide a clear separation of code vs data (in the sense: there is no code since it's just data) and be more performant for the same reasons.